ManagedClientConsoleAppSample/Program.cs

```diff
@@ -78,6 +80,7 @@ private static void ListProjects(AuthenticationHeaderValue authHeader)
                 {
                     Console.WriteLine("\tSuccesful REST call");
                     Console.WriteLine(response.Content.ReadAsStringAsync().Result);
+                    Thread.Sleep(10000);
```

Comment:
why are you adding a sleep here?  If the user wants to stop there, they'll add a breakpoint.  Remove that and then remove the using above.